remoting/rico-remoting-client/src/test/java/dev/rico/client/remoting/impl/TestHttpClientConnector.java

```diff
@@ -105,6 +105,7 @@ public String getHeaderField(String name) {
 
         final ClientModelStore clientModelStore = new ClientModelStore(new DefaultModelSynchronizer(() -> null));
         final HttpClientConnector connector = new HttpClientConnector(getDummyURL(), uiExecutor, backgroundExecutor, clientModelStore, new JsonCodec(), new SimpleExceptionHandler(), Client.getService(HttpClient.class));
+        connector.connect();
```

Comment:
Won't this end in problems since we only have a dummy endpoint? I would assume that an exception will be thrown in a background thread.